Demo/Components/Button/FloatingButton/FloatingButtonDemoView.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import FinniversKit
+
+public class FloatingButtonDemoView: UIView, Tweakable {
```

Comment:
Since it is made for the tweaking options, how about getting Tweaking in the name somehow? Like `TweakingButtonDemoView` or `TweakingFloatingButtonDemoView` if we want to be really specific about what it is and does 🤔

Reply:
It's used in the FINN app as EasterEggButton so it has already multiple uses

Reply:
But it is under `Demo/Components/...` and has `Demo` in the name so this class should not be used outside the FinniversKit demo... are you sure it is this class?

Reply:
`FloatingButtonDemoView` is the Demo view of the component `FloatingButton`.

Reply:
Ah, now I get it, it is the demo of the `FloatingButton`, not a "demo component" to use for showing tweaking options.... 🤦‍♂ 😂

Reply:
I should have probably looked at the entire changeset before commenting 😂

Reply:
Haahah, I was thinking that was the confusion